lib/public.js

```diff
@@ -13,7 +13,8 @@ const
     TRACK = 'track',
     TRACK_ONCE_KEY = '__track_once__',
     WRITE_ACCESS_ERROR = 'This value is not writeable.',
-    ERROR_MESSAGE_ID_AND_EMAIL = 'You must identify a user by email or ID.';
+    ERROR_MESSAGE_ID_AND_EMAIL = 'You must identify a user by email or ID.',
+    PACKAGE_VERSION = require('../package.json').version;
```

Comment:
If we remove the redundant header setting in `publicRequest`, we can remove this import.